UI/obs-app.cpp

```diff
@@ -56,6 +56,9 @@
 
 #ifdef _WIN32
 #include <windows.h>
+#if defined(_M_ARM64)
+	#include <shellapi.h>
+#endif
```

Comment:
Is this even used anywhere? It should also be included through `windows.h` anyway unless `WIN32_LEAN_AND_MEAN` is defined.

Reply:
This header is used by the `ShellExecuteA()` function in `obs-app.cpp`.

https://github.com/obsproject/obs-studio/blob/5854f3b9e5861246ea57dd4a26d3d847a8552c4b/UI/obs-app.cpp#L2926

As `WIN32_LEAN_AND_MEAN` is defined, `shellapi.h` is not included by `windows.h`. By explicitly including `shellapi.h`, we ensure that sources get compiled, preventing compilation errors.

Reply:
I'm not aware of `WIN32_LEAN_AND_MEAN` being defined at this point or this currently causing compilation errors. A cursory check inside Visual Studio shows that `WIN32_LEAN_AND_MEAN` is not defined here. It is defined in other places in the codebase, but not here.

![image](https://github.com/user-attachments/assets/59beedaa-a71f-4586-8ceb-d2cb6e1f1ef8)

Have I misunderstood something?

Reply:
Hi Ryan, The cursory check on WOA device (X-Elite) points that `WIN32_LEAN_AND_MEAN` is defined, so i have added `Shellapi.h` explicitly to resolve compilation issues.

![shellapi](https://github.com/user-attachments/assets/25e5d17a-7937-413a-baeb-1527047ef999)

Reply:
That's strange, but I suppose it's some platform difference that makes this required.